creator/files/models.py

```diff
@@ -28,17 +32,41 @@ def only_printable(string):
     return "".join([s for s in string if s.isprintable()])
 
 
+def extract_config_path(version):
```

Comment:
Where is this used? Why not make it a property of `Version`?

Reply:
Oh you know what this should probably move into the `do-validation` branch where it's actually used. I think I was going to use this function in the tests and then went a different way with tests. I'll move it into `do-validation` and there I can make it part of the `Version` class